app/views/placements/wizards/add_user_wizard/_check_your_answers_step.html.erb

```diff
@@ -1,41 +1,39 @@
-<%# locals: (user: nil, organisation: nil) -%>
-<div class="govuk-width-container">
+<%= content_for :page_title, sanitize(t(".page_title")) %>
+
+<%= form_for(check_your_answers_step, url: current_step_path, method: :put) do |f| %>
+  <%= f.govuk_error_summary %>
+
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <%= form_with(model: user, url: placements_organisation_users_path(organisation), method: :post) do |f| %>
-        <%= f.hidden_field :first_name, value: user.first_name %>
-        <%= f.hidden_field :last_name, value: user.last_name %>
-        <%= f.hidden_field :email, value: user.email %>
-
-        <label class="govuk-label govuk-label--l">
-          <span class="govuk-caption-l"><%= t(".add_user", organisation_name: organisation.name) %></span>
+      <label class="govuk-label govuk-label--l">
+          <span class="govuk-caption-l"><%= t(".add_user", organisation_name: @wizard.organisation.name) %></span>
           <%= t(".check_your_answers") %>
         </label>
 
         <%= govuk_summary_list do |summary_list| %>
           <% summary_list.with_row do |row| %>
             <% row.with_key(text: User.human_attribute_name(:first_name)) %>
-            <% row.with_value(text: user.first_name) %>
+            <% row.with_value(text: @wizard.steps[:user].first_name) %>
             <% row.with_action(text: t(".change"),
-                               href: new_placements_organisation_user_path(organisation, user.as_form_params),
+                               href: back_link_path,
```

Comment:
While `back_link_path` works fine in this case, I think we should probably use:

```suggestion
                               href: step_path(:user),
```

This is a more future-proof approach because it means the change links won't break if another step is ever added to the wizard. It also makes it really obvious for other developers which step the link goes to.